stack/stack.go

```diff
@@ -92,7 +93,9 @@ func fetchYAML(address *url.URL) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	req.Header.Set("User-Agent", fmt.Sprintf("%s %s ", "FaaS-CLI/", version.VersionStr()))
```

Comment:
White space between identifier and version intentional?

Reply:
Also for the terminating space?

Reply:
Good catch :wink: I'll amend the commit